02.calendar/cal.rb

```diff
@@ -0,0 +1,25 @@
+require 'optparse'
+require 'Date'
+
+options = ARGV.getopts('y:', 'm:')
+year  = options["y"].nil? ? Date.today.year  : options["y"].to_i
+month = options["m"].nil? ? Date.today.month : options["m"].to_i
+
+puts "      #{month}月 #{year}"
+puts "日 月 火 水 木 金 土"
+
+first_date = Date.new(year, month, 1)
+last_date = Date.new(year, month, -1)
+first_wday = first_date.wday
+print "   " * first_wday
```

Comment:
```suggestion
print "   " * first_date.wday
```

`first_wday`は1回しか使われないし、`first_date.wday`と比べて可読性が特別上がってるわけでもないので、変数に入れなくてもいいと思いますー。